maquetado/index.html

```diff
@@ -9,6 +9,7 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <img class="image-title" src="./assets/triangle.svg" alt="Icon titulo">
```

Comment:
Try to restrict `img` tags only to content that tells something to the user. In this case, since you're only using this image as a decorator, try to use a pseudo-element. ✌️